sources/packages/backend/libs/sims-db/src/entities/institution-location.model.ts

```diff
@@ -61,4 +61,14 @@ export class InstitutionLocation extends RecordDataModel {
     nullable: false,
   })
   hasIntegration?: boolean;
+
+  /**
+   * Contacts who receive notification email on integration file processing.
+   */
+  @Column("varchar", {
```

Comment:
just wondering we do have `type` in the Column property, is that not working? something like this 
![image](https://github.com/bcgov/SIMS/assets/77353155/9d1f8dbe-881b-478d-807e-b2f635ec17b0)
did u try it?

Reply:
Technically they both are the same for the above context. Reason I went this way is influenced by the example given by typeorm documentation.

https://github.com/typeorm/typeorm/blob/d4607a86723eef07e62e6d7321a07f3ae5ed1f90/test/functional/database-schema/column-types/postgres/entity/Post.ts#L241

For the sake of consistency in our application, I can change it to use type.